docs/esm_tools.rst

```diff
@@ -49,7 +73,7 @@ You can get a pre-generated template to add a new component with::
     Thank you! The new configuration has been saved. Please commit it (and get in touch with the
     esm-tools team if you need help)!
```

Comment:
I changed it in the documentation. But since this is an example of the output that esm_tools will return, it needs to be changed also in the source code. I'll make a note and change it later.